dashboard/src/app/(authenticated)/[tenantId]/(diagram)/components/NodeTypes/UserTask/UserAndGroupAssignmentInfo.tsx

```diff
@@ -1,13 +1,14 @@
+import { WfRunId } from 'littlehorse-client/proto'
 import { FC } from 'react'
 interface UserTaskRunDetailsProps {
-  userGroup: string | number | boolean | Buffer | undefined
-  userId: string | number | boolean | Buffer | undefined
+  userGroup: string | number | boolean | Buffer | undefined | WfRunId
```

Comment:
Can a userGroup be a WfRunId?

Is "explicit" `undefined` required here? Can't it be solved with optional?

```suggestion
  userGroup?: string | number | boolean | Buffer | WfRunId
```

Reply:
Please validate with @Snarr if WfRunId applies in this context.

Reply:
https://github.com/littlehorse-enterprises/littlehorse/blob/25966cd79793d865252ecb84c9052c9a5ff4c489/dashboard/src/app/(authenticated)/%5BtenantId%5D/(diagram)/components/NodeTypes/UserTask/UserTaskDefDetails.tsx#L11

the values passed into this prop are VariableAssignments, which are called within `getVariable`, which does return this exact union. 

This is why the types must match here with undefined and WfRunId.

Reply:
UserId and UserGroup should always be a String I believe, I don't think there's any way for them to be stored in another format on the server. If a UserId or UserGroup is determined by a VariableAssignment, the server converts them to Strings internally.